### PR TITLE
Remove unused `keep_alive_while_idle`

### PR DESCRIPTION
**Pull request by timvisee**
Related to <https://github.com/qdrant/qdrant/issues/1907>.

This removes the `keep_alive_while_idle` setting from our internal gRPC client. It has no effect (because it requires additional settings to enable) and is therefore confusing.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### CODE REVIEW

**Review by ffuugoo — APPROVED** [review]
LGTM. 👌

**Comment by generall** [comment]
does that mean that we don't keep internal connections open anymore? Do we need to establish an internal connection on each request?

**Comment by ffuugoo** [comment]
> does that mean that we don't keep internal connections open anymore? Do we need to establish an internal connection on each request?

See here: https://github.com/qdrant/qdrant/issues/1907#issuecomment-1552836443. 

TL;DR: the option name is confusing, and it has nothing to do with keeping connections open.
